src/debuggability.js

```diff
@@ -764,12 +821,7 @@ var fireGlobalEvent = (function() {
             }
         };
     } else {
-        var globalObject = typeof self !== "undefined" ? self :
```

Comment:
I wonder if this can cause TDZ errors if someone makes a `self` block scoped variable in node for example. Although it doesn't sound like a really problematic case since in practice I suspect this will be a non-issue.